.github/actions/install-codeql/action.yml

```diff
@@ -19,13 +23,25 @@ runs:
         key: codeql-home-${{ inputs.codeql-cli-version }}
   
     - name: Install CodeQL
+      id: install-codeql
       if: steps.cache-codeql.outputs.cache-hit != 'true'
       shell: bash
       env:
         GITHUB_TOKEN: ${{ github.token }}
         CODEQL_HOME: ${{ github.workspace }}/codeql_home
         CODEQL_CLI_VERSION: ${{ inputs.codeql-cli-version }}
       run: |
+        if [ -z "$CODEQL_CLI_VERSION" ]; then
```

Comment:
Maybe it is worth always reading from the `.codeqlversion` file instead of relying on input (then we know that it is always the same CodeQL version being used).